src/main/resources/standardTestCases/empty_string/empty_string.wdl

```diff
@@ -0,0 +1,33 @@
+task hello {
+  command {
+    echo "Hello!"
+  }
+  output {
+    String empty = ""
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+}
+
+task goodbye {
+  String emptyInputString
+  command {
+    echo "${emptyInputString}"
+  }
+  output {
+    String empty = read_string(stdout())
+  }
+  runtime {
+   docker: "ubuntu:latest"
+  }
+}
+
+workflow hello {
+  call hello
```

Comment:
I've been operating on the assumption that the people who added these to CromwellTestKitSpec knew what they were doing